### DISCUSSION TITLE
Show line numbers in file scan

### DISCUSSION THREAD

**Discussion by nivag-9** [projectdiscovery/nuclei#2905] [Q&A]
When scanning files using file-based templates, the line number of the match is not shown. Is there a way to show this or is this a feature that could be added?

**Comment by olearycrew** [comment]
@ehsandeep thoughts here? Should we create an issue?

**Comment by dogancanbakir** [comment]
This discussion closed automatically due to inactivity. Feel free to reopen or start new if still relevant.
